### ISSUE TITLE
Html Page for the app

### ISSUE THREAD

**Issue by kmarrip** [chaithanyaMarripati/goSpotify#8]
- Right now after the users are authorized, they are presented with a simple text file.
- It would be great if the users are greeted with a pleasant html page

**Comment by fastboot** [comment]
Hey, can I work on that page? And can you provide some pointers on what all you are expecting it to have?

**Comment by chaithanyaMarripati** [comment]
hey @fastboot thanks for showing interest in goSpotify

- the app lacks a html page, right now only a text file is sent back https://github.com/chaithanyaMarripati/goSpotify/blob/77e37729daa50888be02d63acd87a8fd01310b10/handler/handler.go#L38
- it would be great if you could create a html folder and add the main html, for now add two place holders name and song currently being played. how you design it using html and css i leave it up to your imagination.
Thanks

**Comment by chaithanyaMarripati** [comment]
just let me know if want to take up this task, i will assign it you. Even a basic html page would suffice. Easy pr @fastboot

**Comment by fastboot** [comment]
Haven't worked much in Go ain't sure how will I pass the name and song variables down to the HTML file rest is no problem. Assign it I'll figure that out

**Comment by chaithanyaMarripati** [comment]
No problem, even if u made a html folder it would suffice
Connect with me on discord if u find any issues
Krishna Chaithanya#3728

**Comment by fastboot** [comment]
@chaithanyaMarripati `docker run -p 8080:8080 gospotify:latest` can't find the .env can you help me set that up?

**Comment by chaithanyaMarripati** [comment]
hey @fastboot any update on this issue, if you are facing any issue please contact me via discord 
Thanks.

**Comment by fastboot** [comment]
Yeah @chaithanyaMarripati couldn't set it up on running the docker run it needed a .env file can you provide me with that?

**Comment by fastboot** [comment]
@chaithanyaMarripati Will work on this can you give me some time, please?

**Comment by chaithanyaMarripati** [comment]
hey @fastboot  if you are facing issues , its ok 
but this project is pretty important to me. i can't this simple issue lie around for ever.
are still working on this(if not will move this someone else ), can i at least see some commits please.

**Comment by koshimagoyal** [comment]
hi @chaithanyaMarripati 
Please assign it to me. I would like to work on it. Also can you please tell what you want in authorized page?

**Comment by kmarrip** [comment]
Hey @koshimagoyal the ask is simple 
Create a html page, with in file css. Plz make the page look more like spotify it self 
Everything else I leave it u

**Comment by koshimagoyal** [comment]
@chaithanyaMarripati can you please help with setup? I installed docker and even created .env file but on build getting permission denied error. Attached error for reference : -
```
~/goSpotify$ docker build -t gospotify:latest .
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/build?buildargs=%7B%7D&cachefrom=%5B%5D&cgroupparent=&cpuperiod=0&cpuquota=0&cpusetcpus=&cpusetmems=&cpushares=0&dockerfile=Dockerfile&labels=%7B%7D&memory=0&memswap=0&networkmode=default&rm=1&shmsize=0&t=gospotify%3Alatest&target=&ulimits=null&version=1": dial unix /var/run/docker.sock: connect: permission denied
```

**Comment by kmarrip** [comment]
hey @koshimagoyal this happens when you dont have the docker running on your unix socket or you dont have enough permissions. 
please do make sure that you have docker running. tbh you dont need docker for this, just create an index.html in templates folder and that would be it.

**Comment by koshimagoyal** [comment]
ok thanks @chaithanyaMarripati !! Also can you please share any link how you want html page to look like? What elements you want in page? Any reference you have.

**Comment by stanipetrosyan** [comment]
I think is enough for now display a simple html page with the information of song and username. In the future with more features, GUI will be improved. 

Obv, this is just my opinion :)

**Comment by kmarrip** [comment]
yes i just wanted a html page to be sent instead text file 
do you want to take this up @StaniPetrosyan ??
any update @koshimagoyal ??

**Comment by koshimagoyal** [comment]
@chaithanyaMarripati sorry not clear with requirements so didn't have any progress. You can assign to someone else.

**Comment by stanipetrosyan** [comment]
Ok, I can take this one :)

**Comment by kmarrip** [comment]
ok @StaniPetrosyan here you go
